app/controllers/heartbeat_controller.rb

```diff
@@ -0,0 +1,51 @@
+require "sidekiq/api"
+
+class HeartbeatController < ActionController::API
+  def ping
```

Comment:
do we need this as well as `healthcheck`? seems redundant

Reply:
I say we keep it for stuff like Frontdoor that pings several times a second.